queries/git_config/injections.scm

```diff
@@ -6,3 +6,26 @@
   value: (string) @injection.content)
   (#any-of? @_name "cmd" "command" "textconv")
   (#set! injection.language "bash"))
+
+(section
+  (section_header
+    (section_name) @_section)
+  (variable
+    value: (string) @injection.content)
+  (#eq? @_section "alias")
```

Comment:
```suggestion
    (section_name) @_section)
  (#eq? @_section "alias")
  (variable
    value: (string) @injection.content)
```

Reply:
I think it's fine for predicates and directives to sit all at the end of the query. Just styling preferences that the formatter can't enforce right now.

Reply:
Yup it's fine, but it'd better in style (although we do not enforce). As in other queries, predicates are not necessarily placed at the end, so I thought it'd be better to follow the style of them.

Reply:
I place them at the end in my queries. ¯\\\_(ツ)\_/¯

Reply:
In general, please avoid stylistic preferences that 1. don't make an observable difference and 2. are not enforced through our formatter. If a convention is worth having, it's worth enforcing; otherwise we'll end up with inconsistent back-and-forth.